### ISSUE TITLE
XGov-Form-Builder summary page has serious Axe accessibility violations.

### ISSUE THREAD

**Issue by ramsharma-prog** [XGovFormBuilder/digital-form-builder#765]
**Describe the bug**
XGov-Form-Builder summary page has serious Axe accessibility violations due to description list item does not have a <dl> parent element

**To Reproduce**

1. Create a summary page on the designer and submit 
2. Run Axe accessibility test.

**Expected behavior**
1. No serious violations

**Screenshots**

<img width="1792" alt="Screenshot 2022-02-24 at 10 49 22" src="https://user-images.githubusercontent.com/80714392/155556174-e7d7777a-e3f9-4ff7-b8db-faf96c44064f.png">

**Repository**

- runner

**Comment by SentientHousePlant** [comment]
Hello! 

I believe this problem could be solved by editing the html template in 

[runner/src/server/views/partials/summary-row.html](https://github.com/XGovFormBuilder/digital-form-builder/blob/main/runner/src/server/views/partials/summary-row.html)

I think there is a div which is preventing the description list from being parsed correctly by the accessibility tool (Axe).

**Comment by jenbutongit** [comment]
markup matches govuk frontend - https://design-system.service.gov.uk/components/summary-list/default/index.html, running a test against axe shows no issues to do with `dl`. I also tested against `/report-a-terrorist/summary` locally, which also doesn't produce the errors. 

upon further debugging, this template was provided
<details>
 <summary>template</summary>
   
  ```
{% extends "base.html" %}
{% from "govuk_frontend_jinja/components/input/macro.html" import govukInput %}
{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
{% block content %}
  <div>
    <form method="POST" action="/">
        {{ form.csrf_token }}
        {{ govukInput({
          'label': {
            'text': "Fund Search",
            'classes': "govuk-label--l",
            'isPageHeading': true
          },
          'id': form.search.id,
          'name': form.search.id,
        })
     }}
     {{ govukButton({
      "text": 'Search', 'preventDoubleClick': True
    }) }}
    </form>
  </div>
 <div>
    <dl class="govuk-summary-list"> <div class="govuk-summary-list"> </dl> <!-- if i close the dl tag in here then it odes not show any serious violation but messes up the layout-->
     </div> <br>
  {% for funds in fund_results %}
        <br> <br>
        <div class="govuk-summary-list__row">
            <dt class="govuk-summary-list__key"> Fund name </dt>
              <dd class="govuk-summary-list__value">
                <a href="{{url_for('discovery_bp.fund_rounds', id=funds['fund_id']) }}"> {{funds['fund_name']}}</a>
              </dd>
        </div>
        <div class="govuk-summary-list__row">
          <dt class="govuk-summary-list__key"> Description </dt>
           <dd class="govuk-summary-list__value">
            <p class="govuk-body"> {{funds['fund_description']}}</p>
           </dd>
     </div>
   {% endfor %}
  {% endblock %}
  ```
</details>

fwiw, this appears to be an override/bespoke code which is missing closing tags which is causing layout issues (as well as possibly confusing axe). it's also unrelated to the OS repo, so closing this issue.
